di_website/spotlight/models.py

```diff
@@ -18,12 +18,17 @@ class Meta():
     country_code = models.CharField(max_length=100, help_text='e.g. UG, KE', default='')
     country_name = models.CharField(max_length=255)
     currency_code = models.CharField(max_length=100, help_text='UGX, KES', default='')
-
+    datasources_description = models.TextField(help_text='A description for data sources section', default='')
+    datasources_url_title = models.TextField(help_text='A title for data sources url link', default='')
+    datasources_url = models.URLField(help_text='A url link to data sources for download', default='')
```

Comment:
I'd rather this wasn't hardcoded ... the slug for the data sources page could change and we'd then have to come change this as well. Since it's an internal page, we'll have to select it.

Reply:
Updated to use both a page chooser and url field